packages/fetchai/skills/http_echo/tests/test_handlers.py

```diff
@@ -44,9 +44,9 @@ class TestHttpHandler(BaseSkillTestCase):
     path_to_skill = PACKAGE_DIR
 
     @classmethod
-    def setup(cls):
+    def setup_class(cls):
```

Comment:
Not all the below seems a class level setup. E.g. the message list `list_of_messages` should stay at `setup` level imo